control_plots_ul.sh

```diff
@@ -42,14 +42,26 @@ if [[ $MODE == "SHAPES" ]]; then
         mkdir -p $shapes_output
     fi
 
-    python shapes/produce_shapes.py --channels $CHANNEL \
-        --directory $NTUPLES \
-        --${CHANNEL}-friend-directory $XSEC_FRIENDS \
-        --era $ERA --num-processes 4 --num-threads 12 \
-        --optimization-level 1 --control-plots \
-        --control-plot-set ${VARIABLES} --skip-systematic-variations \
-        --output-file $shapes_output \
-        --xrootd --validation-tag $TAG
+    if $channels == "mt"; then
```

Comment:
does it make sense to have the mt channel as a condition, as it is also an analysis channel? 
I think it could be useful to have a bash script only for tau id measurement, what do you think?

Reply:
Yes, I can create such a script for tauID explicitly

Reply:
But this this not for tauID measurement but for tauID control plots (to see if sfs make sense)